Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+ruby RUBY_VERSION
+
+gem 'bundler'
```

Comment:
in the context of a gem intended to run on multiple ruby versions, does it make sense to specify in Gemfile? is bundler needed to be specified?